api/v1alpha1/cassandrabackup_types.go

```diff
@@ -32,6 +40,11 @@ type CassandraBackupSpec struct {
 
 	// The name of the CassandraDatacenter to back up
 	CassandraDatacenter string `json:"cassandraDatacenter"`
+
+	// The type of the backup: "full" or "differential"
+	// +kubebuilder:validation:Enum=differential;
```

Comment:
Looks like you left off `full` for the enum values.

Reply:
No worries on the delay, and thanks for spotting the mistake (Recitified now) :)